Cartfile.resolved

```diff
@@ -2,18 +2,21 @@ github "kstenerud/KSCrash" "1.15.8"
 github "getsentry/sentry-swift" "2.1.10"
 github "Alamofire/Alamofire" "4.3.0"
 github "sleroux/Deferred" "35b8927c1b94ce074e10793c57e1f80d0e2227fa"
+github "google/EarlGrey" "1.9.0"
 github "cezheng/Fuzi" "1.0.1"
 github "swisspol/GCDWebServer" "3.3.2"
+github "farhanpatel/JSONSchema.swift" "1c052b83baa8c497e12cde6a8afca0f54574612f"
 github "kif-framework/KIF" "v3.5.1"
+github "kstenerud/KSCrash" "1.15.8"
 github "rs/SDWebImage" "3.7.4"
 github "SnapKit/SnapKit" "3.1.2"
 github "jrendel/SwiftKeychainWrapper" "3.0.1"
+github "skyline75489/SwiftRouter" "2.0.0"
+github "SwiftyJSON/SwiftyJSON" "3.1.4"
```

Comment:
Hmm I don't think we want to include the new commits for libraries we didn't update in the Cartfile. Is there a way we can only include the new SwiftRouter commit? Also, how come we updated the Alamofire version to be 4.3.0?